pdf/pdfcanvas.h

```diff
@@ -33,6 +33,7 @@ class PDFCanvas : public QQuickItem
     Q_PROPERTY( QColor linkColor READ linkColor WRITE setLinkColor NOTIFY linkColorChanged )
     Q_PROPERTY( QColor pagePlaceholderColor READ pagePlaceholderColor WRITE setPagePlaceholderColor NOTIFY pagePlaceholderColorChanged )
     Q_PROPERTY( int currentPage READ currentPage NOTIFY currentPageChanged )
+    Q_PROPERTY(QSizeF maxPageSize READ maxPageSize NOTIFY pageSizesReady)
```

Comment:
I fact no, if I'm not mistaken, the signal corresponds to the property. Maybe the naming scheme is bad. The signal "pageSizesReady" is emitted only when the thread finishes to get the physical sizes of every pages. So the maximum of all the sizes is indeed known at that time only.

In fact, the pageSizesReady signal, is the one that we introduced for the remember page position PR. It is just that now there is a property attached to it, and not only a signal. Thus maybe the naming issue.